components/CheckDocumentation/vacuum/InefficientIndexPhase.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+
+import { CheckDocs, CheckGuidanceProps, CheckTriggerProps } from "../../../util/checks";
+
+const InefficientIndexPhaseTrigger: React.FunctionComponent<CheckTriggerProps> = ({
+  config,
+}) => {
+  const threshold = config.settings["threshold_count"] as number;
+  const runPluralized = threshold === 1 ? 'run' : 'runs';
+  return (
+    <>
+      <p>
+        Detects when autovacuum runs in this server are forced to perform inefficient
+        multiple index scan phases due to limited configured memory, and creates an
+        issue with severity "warning". Triggers when at least <code>{threshold}</code>
+        autovacuum {runPluralized} in the last 24h have had multiple index scan phases.
+        Resolves automatically once fewer than <code>{threshold}</code> autovacuum {runPluralized}
+        in the last 24h have had multiple index scan phases.
+      </p>
+      <p>
+        Ignores situations where configured autovacuum mmemory is already set to the
+        maximum possible value of <code>1GB</code>.
+      </p>
+    </>
+  );
+};
+
+const InefficientIndexPhaseGuidance: React.FunctionComponent<CheckGuidanceProps> = ({
+  urls: { SettingLink },
+}) => {
+  return (
+    <div>
+      <h4>Impact</h4>
+      <p>
+        Autovacuum on affected tables may take significantly longer. VACUUM processes tables
+        and their indexes in phases. To avoid impacting your primary workload, autovacuum
+        is limited to using a certain amount of memory. If this is not enough to process
+        all dead rows, the index scan phase will need to be repeated (possibly several times).
+        This is inefficient in terms of CPU and I/O.
+      </p>
+      <h4>Common Causes</h4>
+      <ul>
+        <li>
+          <h5><code>autovacuum_work_mem</code> set too low</h5>
```

Comment:
I have a slightly mixed feeling about having `<code>` in `<h5>` (not strong feeling). What do you think about saying something like "Autovacuum work memory set too low"?

Reply:
I thought about that too, but it looks okay to me when rendered. I don't mind changing it, though.

Reply:
I don't mind leaving it as is too :P well let's keep it as is